content/modules/dev/ui.editorview.js

```diff
@@ -66,14 +68,23 @@
                     self._doEdit        = false;
                     self._inReplyTo        = 0 ;
                     self._selection        = evt.value.selection;
-                    self._html5range    = evt.value.html5range; 
+                    self._html5range    = evt.value.html5range;
                     self._sel        = null;
                     self._note        = null;
                     model.add("draft", drafts);
                     self._render(id_item, evt.value.suppressFocus);
-                    break;
-                case "reply_thread": 
+<<<<<<< HEAD
```

Comment:
looks like a merge conflict that wasn't properly resolved?

On 3/3/2016 11:57 PM, Kevin Peng wrote:
>
> In content/modules/dev/ui.editorview.js
> <https://github.com/nbproject/nbproject/pull/292#discussion_r54990248>:
>
> >                      self._sel        = null;
> >                      self._note        = null;
> >                      model.add("draft", drafts);
> >                      self._render(id_item, evt.value.suppressFocus);
> > -                    break;
> > -                case "reply_thread": 
> > +<<<<<<< HEAD
>
> Is this supposed to be here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nbproject/nbproject/pull/292/files#r54990248>.
>